examples/helloWorld/README.md

```diff
@@ -413,7 +418,7 @@ uses the map:
 <!-- @countHashes @test -->
 ```
 test 3 == \
-  $(kustomize build $OVERLAYS/staging | grep khk45ktkd9 | wc -l)
+  $(kustomize build $OVERLAYS/staging | grep khk45ktkd9 | wc -l); echo $?
```

Comment:
good idea,
Please put the echo on a new line.

Reply:
Done.